packages/chat-component/src/components/chat-component.ts

```diff
@@ -61,12 +66,18 @@ export class ChatComponent extends LitElement {
   @property({ type: String, attribute: 'data-api-url' })
   apiUrl = chatHttpOptions.url;
 
+  @property({ type: String, attribute: 'data-custom-branding', converter: (value) => value?.toLowerCase() === 'true' })
+  isCustomBranding: boolean = globalConfig.IS_CUSTOM_BRANDING;
```

Comment:
would be good to be consistent with 'useStream' as 'useCustomBranding'

Reply:
I meant more about the already in use "useStream" property just below in this component and to keep to the same standard here and not about porting react convention. But I think hasCustomBranding actually is much better.

Reply:
Agreed, I'd say `hasCustomBranding` would make more sense